alluxio/pom.xml

```diff
@@ -106,7 +106,15 @@
 
         <dependency>
             <groupId>org.alluxio</groupId>
-            <artifactId>alluxio-core-server</artifactId>
+            <artifactId>alluxio-core-common</artifactId>
```

Comment:
this list needs to be updated https://github.com/apache/zeppelin/blob/master/zeppelin-distribution/src/bin_license/LICENSE#L103

if we are not using core-server for example, plus new version numbers